sass/directives/_visual-guides.scss

```diff
@@ -49,72 +53,119 @@
  * TODO: Concerning guides with no steps
  */
 @mixin visual-guide-row ($counter) {
-  @include display(flex);
+  display: block;
   position: relative;
 
-  .visual-guide__content, .visual-guide__image {
-    @include display(flex);
-    width: 50%;
+  &:focus {
+    outline: 0;
+  }
 
-    &:first-of-type {
-      margin-right: $base-spacing-large;
+  .visual-guide__image {
+    display: none;
+    
+    @media only screen and (min-width: $visual-guide-breakpoint-small) {
+      @include align-self(center);
+      @include justify-content(center);     
     }
+  }
+
+  .visual-guide__image--mobile {
+    margin-bottom: $base-spacing;
```

Comment:
👍  Thanks for the heads up. Will keep this in mind next sprint. I did also add a new variable and changed the existing one to xlarge. We will discuss more after this sprint.